game-core/src/main/java/games/strategy/engine/framework/map/download/ContentReader.java

```diff
@@ -28,20 +29,20 @@
 @Log
 @AllArgsConstructor
 // TODO: testing, break up DownloadUtilsTest to test this component individually from DownloadUtils.
```

Comment:
Is this TODO still required now that `DownloadUtilsTest` has been removed?

Reply:
Nope, it is completed now and can be removed  :+1: 

Updated in: 3f15e9aea